cmd/main.go

```diff
@@ -22,22 +22,40 @@ import (
 	"sigs.k8s.io/kubebuilder/v3/pkg/cli"
 	cfgv2 "sigs.k8s.io/kubebuilder/v3/pkg/config/v2"
 	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
+	commonv2 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/common/v2"
```

Comment:
I don’t see a point in complicating the v2 plugin code by breaking it up. Just leave it as-is.

Reply:
I agree. That was my first idea here too.

Reply:
I prefer it to be the samne as v3, so that when we have to fix a bug in v2 we can find it in the same place as v3, and not the common part for v3 but the full plugin for v2.

Reply:
Hi @Adirio, 

See how it is now. I updated the first comment as well to clarify. IMO it is much simpler and understandable. 
Note that:
- The new config-base plugin does not need to have the same version of the language ones. 
- And we have no motivation to want to do that with `golang/v2`.  (That only exist to ensure the backwards compability)

Reply:
I think that we are all on the same page now. However, if not. please feel free to re-open this one or ping me out.